tdrs-backend/tdpservice/search_indexes/tasks.py

```diff
@@ -0,0 +1,37 @@
+"""Shared celery search_indexes tasks for beat."""
+
+from __future__ import absolute_import
+import time
+from tdpservice.users.models import User
+from celery import shared_task
+from tdpservice.core.utils import log
+from django.core.management import call_command
+
+
+@shared_task
+def reindex_elastic_documents():
+    """Run the django-elasticsearch-dsl management command to reindex the elastic instance."""
+    system_user, created = User.objects.get_or_create(username='system')
+    if created:
+        log('Created reserved system user.')
+
+    start = time.time()
+    log('Starting elastic reindex.', {
+        'user_id': system_user.pk,
+        'object_id': None,
+        'object_repr': 'Exception in search_index --rebuild'
+    })
+
+    call_command('search_index', '--rebuild', '-f', '--parallel')
```

Comment:
One thing I thought of post OH today. Should we also add the option to parametrize the models upon which we re-index? That will save us a bunch of time if we could somehow figure out how to check when a model's schema has been changed.